### PR TITLE
Add large payload testing page to the sandbox.

### PR DESCRIPTION
**Pull request by jfkhoury**
## Description

We need a page where we can test beacon calls and large payloads being send to the edge.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All tests pass and I've made any necessary test changes.
- [ x ] I have run the Sandbox successfully.

### CODE REVIEW

File: sandbox/src/LargePayload.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+export default function LargePayload() {
+
+  const makePayload = size => () => {
+    const payload = new Uint8Array(size * 1024);
+    window.alloy("event", {
+      beacon: true,
```
Comment (jonsnyder) [inline sandbox/src/LargePayload.js]: `beacon: true` doesn't actually do anything.

Comment (jfkhoury) [reply sandbox/src/LargePayload.js]: :) That's what happens when I copy/paste :)

I think from the Links page. I'll clean it up.
